Controller/ControllerTrait.php

```diff
@@ -12,17 +12,23 @@
 namespace FOS\RestBundle\Controller;
 
 use FOS\RestBundle\View\View;
-use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use FOS\RestBundle\View\ViewHandlerInterface;
 use Symfony\Component\HttpFoundation\Response;
 
 /**
- * Base Controller for Controllers using the View functionality of FOSRestBundle.
+ * Trait for Controllers using the View functionality of FOSRestBundle.
  *
  * @author Benjamin Eberlei <kontakt@beberlei.de>
+ * @author Lukas Kahwe Smith <smith@pooteeweet.org>
  */
-abstract class FOSRestController extends Controller
+trait ControllerTrait
```

Comment:
Finally, I'm wondering if it's not too complicated to explain if there are traits... 
Can't we just check if ``$this`` is an instance of ``ContainerAwareInterface`` or if ``$this->viewHandler`` is defined and otherwise throw an exception?